package.json

```diff
@@ -9,8 +9,8 @@
     "test-watch": "./node_modules/.bin/jest --watch",
     "test-debug": "node --inspect-brk ./node_modules/jest/bin/jest.js --runInBand --watch",
     "test-e2e": "cross-env NODE_ENV=test BABEL_DISABLE_CACHE=1 mocha --retries 2 --compilers js:@babel/register --require ./test/setup.js ./test/e2e.js",
-    "lint": "./node_modules/.bin/eslint app --ext .js --ext .jsx && ./node_modules/.bin/stylelint 'app/**/*.css' --ip 'app/dist'",
-    "lint-fix": "./node_modules/.bin/eslint app --ext .js --ext .jsx --fix && ./node_modules/.bin/stylelint 'app/**/*.css' --ip 'app/dist' --fix",
+    "lint": "./node_modules/.bin/eslint app --ext .js --ext .jsx && ./node_modules/.bin/eslint test --ext .js && ./node_modules/.bin/stylelint 'app/**/*.css' --ip 'app/dist'",
+    "lint-fix": "./node_modules/.bin/eslint app --ext .js --ext .jsx --fix && ./node_modules/.bin/eslint test --ext .js --fix && ./node_modules/.bin/stylelint 'app/**/*.css' --ip 'app/dist' --fix",
```

Comment:
Unrelated to your PR, but since you're touching this: You can drop the `--ip app/dist` since we have the global ignore paths in the .eslintignore file.

Reply:
done